dist/background.js

```diff
@@ -0,0 +1,468 @@
YWRlKGRiLCBvbGRWZXJzaW9uLCBuZXdWZXJzaW9uLCB0cmFuc2FjdGlvbikge1xuICAgICAgY29uc29sZS5sb2coXCJjcmVhdGluZyBzdG9yZVwiKTtcbiAgICAgIGNvbnN0IHN0b3JlID0gZGIuY3JlYXRlT2JqZWN0U3RvcmUoc3RvcmVOYW1lKTtcbiAgICB9LFxuICAgIGJsb2NrZWQoKSB7XG4gICAgICBjb25zb2xlLmxvZyhcImJsb2NrZWRcIik7XG4gICAgfSxcbiAgICBibG9ja2luZygpIHtcbiAgICAgIGNvbnNvbGUubG9nKFwiYmxvY2tpbmdcIik7XG4gICAgfSxcbiAgICB0ZXJtaW5hdGVkKCkge1xuICAgICAgY29uc29sZS5sb2coXCJ0ZXJtaW5hdGVkXCIpO1xuICAgIH1cbiAgfSk7XG5cbiAgcmV0dXJuIGRiO1xufVxuXG5hc3luYyBmdW5jdGlvbiBzZXRWYWwoc3RvcmVOYW1lLCB2YWwsIGtleSkge1xuICBsZXQgZGIgPSBhd2FpdCBnZXREQihzdG9yZU5hbWUpO1xuICBjb25zdCB0eCA9IGRiLnRyYW5zYWN0aW9uKHN0b3JlTmFtZSwgXCJyZWFkd3JpdGVcIik7XG4gIGNvbnN0IHN0b3JlID0gYXdhaXQgdHguc3RvcmU7XG4gIGNvbnN0IHZhbHVlID0gYXdhaXQgc3RvcmUucHV0KHZhbCwga2V5KTtcbiAgYXdhaXQgdHguZG9uZTtcbn1cblxuYXN5bmMgZnVuY3Rpb24gZ2V0VmFsKHN0b3JlTmFtZSwga2V5KSB7XG4gIGxldCBkYiA9IGF3YWl0IGdldERCKHN0b3JlTmFtZSk7XG4gIGNvbnN0IHZhbHVlID0gYXdhaXQgZGIuZ2V0KHN0b3JlTmFtZSwga2V5KTtcbiAgcmV0dXJuIHZhbHVlO1xufVxuXG5hc3luYyBmdW5jdGlvbiBxdWVyeShzdG9yZU5hbWUsIGtleSwgcG9ydCkge1xuICBsZXQgdmFsdWUgPSBhd2FpdCBnZXRWYWwoc3RvcmVOYW1lLCBrZXkpO1xuXG4gIGlmICghdmFsdWUpIHtcbiAgICB2YWx1ZSA9IGZhbHNlO1xuICAgIHNldFZhbChzdG9yZU5hbWUsIHZhbHVlLCBrZXkpO1xuICB9XG5cbiAgcG9ydC5wb3N0TWVzc2FnZSh7IGlkOiB7IGtleToga2V5LCB2YWx1ZTogdmFsdWUgfSB9KTtcbn1cblxuYXN5bmMgZnVuY3Rpb24gdG9nZ2xlKHN0b3JlTmFtZSwga2V5LCBwb3J0KSB7XG4gIGxldCB2YWx1ZSA9IGF3YWl0IGdldFZhbChzdG9yZU5hbWUsIGtleSk7XG4gIHZhbHVlID0gIXZhbHVlO1xuICBzZXRWYWwoc3RvcmVOYW1lLCB2YWx1ZSwga2V5KTtcbiAgcG9ydC5wb3N0TWVzc2FnZSh7IGlkOiB7IGtleToga2V5LCB2YWx1ZTogdmFsdWUgfSB9KTtcbn1cblxuYXN5bmMgZnVuY3Rpb24gc2V0VHJ1ZShzdG9yZU5hbWUsIGtleSwgcG9ydCkge1xuICBzZXRWYWwoc3RvcmVOYW1lLCB0cnVlLCBrZXkpO1xuICBwb3J0LnBvc3RNZXNzYWdlKHsgaWQ6IHsga2V5OiBrZXksIHZhbHVlOiB0cnVlIH0gfSk7XG59XG5cbmNocm9tZS5ydW50aW1lLm9uQ29ubmVjdC5hZGRMaXN0ZW5lcihmdW5jdGlvbihwb3J0KSB7XG4gIGNvbnNvbGUubG9nKFwiY29ubmVjdGVkIHRvOiBcIiwgcG9ydC5uYW1lKTtcbiAgY29uc29sZS5hc3NlcnQocG9ydC5uYW1lID09IFwiYmFuZGNhbXBsYWJlbHZpZXdcIik7XG5cbiAgLy8gZ2V0IHZhbHVlcyBvZiBpbml0aWFsXG4gIHBvcnQub25NZXNzYWdlLmFkZExpc3RlbmVyKGZ1bmN0aW9uKG1zZykge1xuICAgIGlmIChtc2cucXVlcnkpIHF1ZXJ5KFwicHJldmlld3NcIiwgbXNnLnF1ZXJ5LCBwb3J0KTtcbiAgICBpZiAobXNnLnRvZ2dsZSkgdG9nZ2xlKFwicHJldmlld3NcIiwgbXNnLnRvZ2dsZSwgcG9ydCk7XG4gICAgaWYgKG1zZy5zZXRUcnVlKSBzZXRUcnVlKFwicHJldmlld3NcIiwgbXNnLnNldFRydWUsIHBvcnQpO1xuICB9KTtcbn0pO1xuXG5jaHJvbWUucnVudGltZS5vbkluc3RhbGxlZC5hZGRMaXN0ZW5lcihmdW5jdGlvbigpIHtcbiAgZnVuY3Rpb24gaXNFbXB0eShvYmopIHtcbiAgICByZXR1cm4gT2JqZWN0LmtleXMob2JqKS5sZW5ndGggPT09IDA7XG4gIH1cbiAgLy8gdXBncmFkZSBvbGQgc3RvcmFnZVxuICBjb25zdCBzdG9yZU5hbWUgPSBcInByZXZpZXdzXCI7XG4gIGNocm9tZS5zdG9yYWdlLnN5bmMuZ2V0KHN0b3JlTmFtZSwgZnVuY3Rpb24ocmVzdWx0KSB7XG4gICAgdHJ5IHtcbiAgICAgIGlmICghaXNFbXB0eShyZXN1bHQpKSB7XG4gICAgICAgIHJlc3VsdFtzdG9yZU5hbWVdLmZvckVhY2goZnVuY3Rpb24oaXRlbSwgaW5kZXgpIHtcbiAgICAgICAgICBzZXRWYWwoc3RvcmVOYW1lLCB0cnVlLCBpdGVtKTtcbiAgICAgICAgfSk7XG4gICAgICB9XG4gICAgfSBjYXRjaCAoZSkge1xuICAgICAgY29uc29sZS5lcnJvcihlKTtcbiAgICB9XG4gIH0pO1xufSk7XG4iXSwic291cmNlUm9vdCI6IiJ9
```

Comment:
I know this is from `webpack` but any idea why it's here?
Maybe there is a "remove comments" option for webpack that could clean this up?

Reply:
Webpack does have a comment removal option, but this is something different - it's a source map, which allows browsers to run this JS file in its built format and debug in its original format. It _can_ be removed but it doesn't actively hurt anything, and it provides value when debugging.

There is an elephant in this pattern: `dist/` isn't typically tracked in VC, it's built on arrival or for packaging. I think `dist/` is included now so that it's easier to deploy (ship to app store), but if you decide to use a workflow to ship to Chrome then `dist/` should probably be gitignored.

Doesn't matter to me either way!

Reply:
Ohhhh! Wow I was not paying attention this monring.

Yeah, so actually I wasn’t planning on adding these files to the repo
because their source of truth is generated from the files in `./src`.

I have a deploy step in the npm configuration file that handles just in
time “compilation” before submitting a release.

Please remove `./dist` from this PR and add to ignore.